package.json

```diff
@@ -25,12 +25,6 @@
       "pre-commit": "tsdx lint"
     }
   },
-  "prettier": {
```

Comment:
ahhh, did not know these were in package.json. I thought they were only in `.prettierrc.json`. Perhaps we should check the other repos for this. Maybe this is why there were/are conflicts...

Reply:
all checked there is an easlint rules in id-demo that should be removed

Reply:
please make a PR for that as well